src/services/sam/projects/projects.ts

```diff
@@ -0,0 +1,146 @@
+import {Logger} from "../../../utils/logger";
+import {AxiosInstance, AxiosResponse} from "axios";
+import {
+  GetMemberProjectsResponse,
+  CreateProjectResponse,
+  GetProjectResponse,
+  UpdateProjectResponse,
+  GetProjectMembersResponse,
+  CreateProjectKeyResponse,
+  RemoveProjectKeyResponse,
+  AddPermissionToKeyResponse, RemovePermissionFromKeyResponse, GetPermissionFromKeyResponse
```

Comment:
Any reason why these 3 don't get their own lines?

Reply:
nope ctrl alt L didnt work for some reason i can fix that